### ISSUE TITLE
elasticsearch.service fails to start when running init

### ISSUE THREAD

**Issue by bm1391** [aol/moloch#714]
After downloading the software and running:

`/data/moloch/db/db.pl http://localhost:9200 init`

I get the response:

> Couldn't GET http://localhost:9200/_cluster/health  the http status code is 500 are you sure elasticsearch is running/reachable? at /data/moloch/db/db.pl line 124.

Running `sudo systemctl status elasticsearch.service`

Returns:

`● elasticsearch.service - Moloch Demo Elasticsearch
   Loaded: loaded (/etc/systemd/system/elasticsearch.service; enabled; vendor preset: enabled)
   Active: inactive (dead) (Result: exit-code) since Wed 2017-07-12 14:06:43 EDT; 4min 56s ago
  Process: 1666 ExecStart=/bin/su -s /bin/sh -c exec "$0" "$@" > /dev/null nobody -- /data/moloch/elasticsearch/bin/elasticsearch -Des.default.path.conf=/data/moloch/etc (code=exited
 Main PID: 1666 (code=exited, status=127)

Jul 12 14:06:43 monroe systemd[1]: elasticsearch.service: Unit entered failed state.
Jul 12 14:06:43 monroe systemd[1]: elasticsearch.service: Failed with result 'exit-code'.
Jul 12 14:06:43 monroe systemd[1]: elasticsearch.service: Service hold-off time over, scheduling restart.
Jul 12 14:06:43 monroe systemd[1]: Stopped Moloch Demo Elasticsearch.
Jul 12 14:06:43 monroe systemd[1]: elasticsearch.service: Start request repeated too quickly.
Jul 12 14:06:43 monroe systemd[1]: Failed to start Moloch Demo Elasticsearch.`

Any help would be appreciated.. thanks

**Comment by awick** [comment]
Please fill out the template with bug reports.

OS kind/version
Moloch kind/version
How did you download

**Comment by bm1391** [comment]
Ubuntu 16.04.2 xenial

Moloch version/Installation method:

`sudo dpkg -i moloch_0.17.0-1_amd64.deb
sudo /data/moloch/bin/Configure
wget https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/deb/elasticsearch/2.4.0/elasticsearch-2.4.0.deb
sudo dpkg -i elasticsearch-2.4.0.deb
sudo systemctl enable elasticsearch.service`

This is where I get problems, running:

`sudo systemctl start elasticsearch.service`

**Comment by awick** [comment]
can you try a newer version?

**Comment by bm1391** [comment]
Sure, is there a great way to remove all of Moloch's dependencies or should I try to manually remove?

**Comment by awick** [comment]
if you just install the newer version it will replace them, or maybe i dont understand the question

**Comment by bm1391** [comment]
Nope, that answers it, Thanks.

**Comment by bm1391** [comment]
Hmm, still getting the same message when running:
enable and start elastic search and then:

`sudo systemctl status elasticsearch.service`

**Comment by awick** [comment]
what version did you try? (please always include in bug reports anywhere)

**Comment by bm1391** [comment]
Moloch 0.19.0-1
2017-07-11

**Comment by awick** [comment]
are you installing elasticsearch yourself or is the Configure script doing it for you?

**Comment by bm1391** [comment]
Beforehand I ran the Configure script and answered [no] and installed elasticsearch myself.

This time I did the Demo environment and now the service is running fine. Must have been Elasticsearch causing the problem

`● elasticsearch.service - Moloch Demo Elasticsearch
   Loaded: loaded (/etc/systemd/system/elasticsearch.service; enabled; vendor preset: enabled)
   Active: active (running) since Wed 2017-07-12 14:55:43 EDT; 7s ago
 Main PID: 2857 (su)
    Tasks: 0
   Memory: 404.0K
      CPU: 4ms
   CGroup: /system.slice/elasticsearch.service
           ‣ 2857 /bin/su -s /bin/sh -c exec "$0" "$@" > /dev/null nobody -- /data/moloch/elasticsearch/bin/elasticsearch -Des.default.path.conf=/data/moloch/etc

Jul 12 14:55:43 monroe systemd[1]: Started Moloch Demo Elasticsearch.
Jul 12 14:55:43 monroe su[2857]: Successful su for nobody by root
Jul 12 14:55:43 monroe su[2857]: + ??? root:nobody
Jul 12 14:55:43 monroe su[2857]: pam_unix(su:session): session opened for user nobody by (uid=0)
Jul 12 14:55:46 monroe systemd[1]: Started Moloch Demo Elasticsearch.
`
